packages/resolve-es-memory/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "resolve-es-memory",
+  "version": "0.0.1",
+  "description": "",
+  "main": "./dist/index.js",
+  "scripts": {
+    "prepublish": "babel --out-dir ./dist ./src",
+    "test": "mocha",
+    "testw": "mocha -w"
+  },
+  "author": "",
+  "license": "MIT",
+  "devDependencies": {
+    "chai": "^3.5.0",
+    "mocha": "^3.2.0",
+    "mock-fs": "^4.2.0",
+    "babel-preset-es2015": "^6.24.1",
```

Comment:
This and the following one are not require - *babel* related dev packages are listed in the top level *package.json*.